test_project/store/src/main/java/test/sergeyvalyushko/store/SaxExample.java

```diff
@@ -0,0 +1,49 @@
+package test.sergeyvalyushko.store;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+public class SaxExample {
+    private static Map<String, String> sort = new HashMap<>();
+    public Map<String, String> config () throws ParserConfigurationException, SAXException, IOException {
```

Comment:
not very elegant implementation, but it is not necessary to fix it if you dont want